### ISSUE TITLE
Allow ⌘-q and ⌘-w shortcuts without mouse or confirmation

### ISSUE THREAD

**Issue by codyaweber** [hankinsoft/SQLPro#977]
2 small QoL suggestions. I don't like it when applications inhibit standard keyboard shortcuts, eg `⌘-q` or `⌘-w`, forcing me to use the mouse instead. When SQL Pro Studio is opened, the start screen with a list of my saved connections cannot be closed with `⌘-q` - I have to hit `Cancel` and then quit. Would be nice if that was not the case. 

Additionally, I almost never save the changes I make in query windows. If I do make a change, I am forced me to confirm I want to delete the tab. That's obviously understandable to avoid losing work, but it'd be nice to have an option to allow skipping the confirmation to make tab management faster.

Environment
 - Device: 2022 14" M2 MBP
 - OS: Sonoma 14.4.1
 - SQLPro app Version: Version 2024.21 (Build 111093)
- Installation source: Setapp
- Target database server: Any

**Comment by hankinsoft** [comment]
Thats for the suggestion. I actually thought this option already existed, but I'm not seeing it anywhere. I'll take a look into getting something added for it.

**Comment by samheuck** [comment]
Yeah I actually came here to request the same feature @codyaweber! I was happy to see someone else had also requested it.

@hankinsoft I just wanted to say that I really enjoy using SQLPro Studio. Your work on this tool is impressive - thank you very much for this excellent tool!

If there's anything I can do to help out or contribute on this, I'd be glad to. That said I completely understand if you prefer to manage feature requests internally.

Cheers

**Comment by hankinsoft** [comment]
For the next build (should be available sometime next week).

First, command-q will now work on the connections window. You will no longer have to hit cancel first.

Next, there is a new 'Window & Session behaviour' section in the general tab. It contains the following options:

  Shutdown action
  - Save state and restore on next launch (default) - windows and tabs come back next launch, including unsaved scratch tabs. No prompt on close or  quit.                                                                                                                                              
  - Clean shutdown - everything closes; next launch starts fresh (no tabs, no open connections). On launch the connection window will be visible.

  On window close or shutdown: prompt about unsaved changes (only when Shutdown action = Clean)                                                      
  - On (default) - get the classic "Review / Discard / Cancel" dialog before unsaved scratch tabs are discarded.                                     
  - Off - silent discard, no prompt.

  On tab close: prompt about unsaved changes (independent of Shutdown action)
  - On (default) - clicking the X on a scratch tab asks before discarding it.
  - Off - close-tab discards silently.

  Tabs opened from a .sql file on disk are autosaved continuously and never trigger any of these prompts. The toggles only apply to scratch tabs.

<img width="1012" height="662" alt="Image" src="https://github.com/user-attachments/assets/1226df23-49e4-4a7a-bbf6-853f6dea8f52" />
